reinforcement-learning/dqn.py

```diff
@@ -0,0 +1,317 @@
+import gym
+import time
+import math
+import sigopt
+import numpy as np
+import tensorflow as tf
+
+
+# You can find your API token at https://sigopt.com/user/tokens
```

Comment:
On second thought maybe we should link to https://sigopt.com/docs/overview/authentication instead